web/src/portico/landing-page.js

```diff
@@ -119,78 +172,16 @@ const apps_events = function () {
 };
 
 const events = function () {
-    // get the location url like `zulip.com/features/`, cut off the trailing
-    // `/` and then split by `/` to get ["zulip.com", "features"], then
-    // pop the last element to get the current section (eg. `features`).
-    const location = window.location.pathname.replace(/\/$/, "").split(/\//).pop();
-
-    $(`[data-on-page='${CSS.escape(location)}']`).addClass("active");
-
-    $("body").on("click", (e) => {
-        const $e = $(e.target);
-
-        if ($e.is("nav ul .exit")) {
-            $("nav ul").css("transform", "translate(-350px, 0)");
-            // See https://ishadeed.com/article/layout-flickering/ for
-            // more context as to why the following timeout is important.
-            setTimeout(() => {
-                $("nav ul").removeClass("show");
-                $("nav ul").css("transform", "");
-                $("body").removeClass("noscroll");
-            }, 500);
-        }
-
-        if ($("nav ul.show") && !$e.closest("nav ul.show").length && !$e.is("nav ul.show")) {
-            $("nav ul").removeClass("show");
-            $("body").removeClass("noscroll");
-        }
-    });
-
-    $(".hamburger").on("click", (e) => {
-        $("nav ul").addClass("show");
-        $("body").addClass("noscroll");
-        e.stopPropagation();
-    });
-
     if (path_parts().includes("apps")) {
         apps_events();
     }
+
+    if (path_parts().includes("hello")) {
+        hello_events();
+    }
 };
 
 $(() => {
-    // Initiate the bootstrap carousel logic
-    $(".carousel").carousel({
-        interval: false,
-    });
```

Comment:
After this commit, we can add a follow-up that removes all the carousel code from `web/third/bootstrap/css/bootstrap.css` and `web/third/bootstrap/js/bootstrap.js`, since this is the only place we were using that Bootstrap feature.

Reply:
Yeah.

Reply:
Added a couple commits to this PR doing this so that we can resolve this comment.